site/class-1-bar/readme.md

```diff
@@ -1,7 +0,0 @@
-# Bar Chart (Class 1) and Readme (Class 2)
```

Comment:
This looks great, but you’re removing our readme! 😢

Reply:
I Noticed! Then moved all files to a maridjuice folder. Now it's correct... correct?

Reply:
Nope, you're still proposing to remove this file. Either try again, or try readding this file with its original content!